include/freerdp/server/proxy/proxy_config.h

```diff
@@ -109,6 +109,9 @@ extern "C"
 		size_t PrivateKeyPEMLength;
 
 		wIniFile* ini;
+
+		/* target continued */
+		INT32 TargetTlsSecLevel;
```

Comment:
The `TlsSecLevel` is an `UINT32` (using this type also eliminates introduction of these new _int32 macros/functions)

Reply:
we need to support 'not available' for this option too (aka we don´t touch the default)

Reply:
So why don't we just set it to 1 by default (like we also do in `pf_server_config_dump`) and avoid checking for set/not set?

Reply:
ok, that we can do.